internal/orchestrator/committer.go

```diff
@@ -15,8 +15,8 @@ import (
 	"github.com/thirdweb-dev/indexer/internal/storage"
 )
 
-const DEFAULT_COMMITTER_TRIGGER_INTERVAL = 250
-const DEFAULT_BLOCKS_PER_COMMIT = 10
+const DEFAULT_COMMITTER_TRIGGER_INTERVAL = 2000
+const DEFAULT_BLOCKS_PER_COMMIT = 1000
```

Comment:
Just to share the idea, these defaults should actually be configured in Viper in the future. Would move all the config to one place.

Not necessary to do it now, but in the larger rework of the config

Reply:
until we do that let's have reasonable defaults